src/features/common/Layout/Footer/index.tsx

```diff
@@ -90,7 +89,6 @@ export default function Footer(): ReactElement | null {
       }
       if (localStorage.getItem('language')) {
         const langCode = localStorage.getItem('language');
-        if (langCode) setLanguage(langCode);
       }
```

Comment:
This code is unused as well now.

Reply:
This was missed, resolved here > [9dd8228](https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1890/commits/9dd822814f6c3cfa51524d820a815478f2d632c4)